src/main/charts/bitbucket/templates/rbac.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and (.Values.bitbucket.clustering.enabled) (or .Values.serviceAccount.clusterRole.create .Values.serviceAccount.role.create) -}}
+apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.serviceAccount.clusterRole.create }}
+kind: ClusterRole
+{{- else }}
+kind: Role
+{{- end }}
+metadata:
+  name: {{ template "bitbucket.clusterRoleName" . }}
+  labels:
+  {{- include "common.labels.commonLabels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - endpoints
+      - pods
+{{- if .Values.serviceAccount.clusterRole.create }}
```

Comment:
Can we refer to the hazelcast documentation, why this is needed?

Reply:
I have dropped the link in the PR description https://github.com/hazelcast/hazelcast-kubernetes#zone-aware or do you mean it'd be nice to have it in the comment too?

Reply:
Updated PR description with a link to Hazelcast readme and a log snippet.